docs/SMCWG-charter.md

```diff
@@ -77,21 +77,27 @@ capitalized terms in this Charter.
 The authorized scope of the SMCWG SHALL be to discuss, adopt, and maintain
 policies, frameworks, and sets of standards related to the issuance and
 management of S/MIME certificates by third-party CAs under a publicly trusted
-root, limited as follows:
+root.
 
-- Baseline verification of control over email addresses, including those used
-  by a natural person or a legal entity, or used by automated systems such as
-  for mailing lists
-- Identity validation for natural persons and legal entities
+The primary deliverable shall be scoped as follows:
+
+- Verification of control over email addresses
 - Key management and certificate lifecycle (subject to coordination with other
   Forum CWGs to ensure consistency and avoid redundancy)
 - Certificate profiles for S/MIME certificates and Issuing CA certificates
   (including the appropriateness of extensions and when those extensions should
   be present)
 - CA operational practices, physical/logical security, etc.
 
-The SMCWG SHALL exercise caution to ensure that its work product does not impede
```

Comment:
I think I'd just change this to something like "It is important that the SMCWG exercise caution ...".  It's really an advisory note; it doesn't make sense as a requirement and shouldn't be one.

Note that it is saying something slightly different from what you replaced it with.

Reply:
@timfromdigicert Could you be more precise? I'm not sure how to address this feedback, because I may be misunderstanding it.

Your proposed "It is important" would still leave it ambiguous as to what the thing that is important is.

Reply:
The important thing is not impeding work on other types of certificates.  An example of this would be unnecessary divergences from other certificate profiles, for example.  I don't think any such statement can be made operationally useful as a requirement.  I think the only purpose of stating it is as documentation of an aspirational goal that people can point back to when they feel necessary to do so.

Reply:
@timfromdigicert I'm not sure how "divergence from other certificate profiles" is "impeding", which I think supports the goal for this being clear. Also, "unnecessarily" I think can be seen somewhat subjectively. For example, if the majority of CAs issued S/MIME certificates for 5 years, and an SMCWG profile set it for 3 years, I think some members might want to claim the lifetime divergence is "unnecessary" while others might see it as "necessary"

I'd like to try to avoid those ambiguities, which I think come up with subjective phrases like "unnecessarily" or "impede". I totally understand the aspirational aspect, but we should try to figure out how to have that well-stated, right?

If the goal is to say "The SMCWG SHOULD consider existing practices when developing these profiles" (handwave), then we can find some language to tighten that up.  My worry is that "consider" is still aspirational, and if there are disagreements, I worry that it will be seen as a Charter disagreement / Code of Conduct issue, as you suggested. Which is why it's important to try to flesh these out now, during the charter, especially when we know past confusion or disagreements.

Reply:
I don't think this badly written sentence is worth this much effort to save, so I now support deleting it.  It's good to avoid conflicts, but doing it in the charter this way is probably a bad idea.